opentelemetry-instrumentation/README.rst

```diff
@@ -63,10 +63,10 @@ Well known trace exporter names:
     - jaeger
     - opencensus
     - otlp
-    - otlp_span
+    - otlp_proto_grpc_span
     - zipkin
 
-``otlp`` is an alias for ``otlp_span``.
+``otlp`` is an alias for ``otlp_proto_grpc_span``.
```

Comment:
Is this true?

Reply:
yep, currently the distro does this:
https://github.com/open-telemetry/opentelemetry-python/blob/c61a7126445f57748af8f7ffd1b7fc0cdadd5476/opentelemetry-distro/src/opentelemetry/distro/__init__.py#L64-L66